docs/content/integrations/dbt/reference.mdx

```diff
@@ -532,6 +532,39 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
 ---
 
+## Emit dbt column schema as materialization metadata <Experimental />
```

Comment:
Does this heading need the word "dbt" in it?

Reply:
Nope, I removed it. Column schema is not specific to dbt.